.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,268 @@
+# This code-samples file is used by the MeiliSearch documentation
+# Every example written here will be automatically fetched by
+# the documentation on build
+# You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
+---
+get_one_index_1: |-
+  client.getIndex("movies");
+list_all_indexes_1: |-
+  client.getAllIndexes();
+create_an_index_1: |-
+  client.createIndex("movies", "movie_id");
+update_an_index_1: |-
+  client.updateIndex("movie_review", "movie_review_id");
+delete_an_index_1: |-
+  client.deleteIndex("movies");
+get_one_document_1: |-
+  client.index("movies").getDocument("25684");
+get_documents_1: |-
+  client.index("movies").getDocuments();
+add_or_replace_documents_1: |-
+  client.index("movies").addDocuments("[{
+    "id": 287947,
+    "title": "Shazam",
+    "poster": "https://image.tmdb.org/t/p/w1280/xnopI5Xtky18MPhK40cZAGAOVeV.jpg",
+    "overview": "A boy is given the ability to become an adult superhero in times of need with a single magic word.",
+    "release_date": "2019-03-23"
+  }]");
+add_or_update_documents_1: |-
+  client.index("movies").updateDocuments("[{
+    "id": 287947,
+    "title": "Shazam ⚡️",
+    "genres": "comedy"
+  }]");
+delete_all_documents_1: |-
+  client.index("movies").deleteAllDocuments();
+delete_one_document_1: |-
+  client.index("movies").deleteDocument("25684");
+delete_documents_1: |-
+  client.index("movies").deleteDocuments(Arrays.asList(new String[]
+{
+"23488",
+"153738",
+"437035",
+"363869"
+}));
+search_post_1: |-
+  client.index("movies").search("American ninja");
+get_update_1: |-
+  client.index("movies").getUpdate(1).getStatus();
+get_all_updates_1: |-
+  for(Update update: client.index("movies").getUpdates())
+  {
+    update.getStatus();
+  }
+get_keys_1: |-
+  //Not yet implemented
+get_settings_1: |-
+  client.index("movies").getSettings();
+update_settings_1: |-
+  //Unsure
+reset_settings_1: |-
+  client.index("movies").resetSettings();
+get_synonyms_1: |-
+  client.index("movies").getSynonyms();
+update_synonyms_1: |-
+  //Unsure
+reset_synonyms_1: |-
+  //Not yet implemented
+get_stop_words_1: |-
+  client.index("movies").getStopWords();
+update_stop_words_1: |-
+  client.index("movies").setStopWords(new String[] {"of", "the", "to"});
+reset_stop_words_1: |-
+  //Not yet implemented
+get_ranking_rules_1: |-
+  client.index("movies").getRankingRules();
+update_ranking_rules_1: |-
+  client.index("movies").setRankingRules(new String[]
+  {
+  "words",
+  "typo",
+  "proximity",
+  "attribute",
+  "sort",
+  "exactness",
+  "release_date:asc",
+  "rank:desc"
+  });
+reset_ranking_rules_1: |-
+  //Not yet implemented
+get_distinct_attribute_1: |-
+  client.index("movies").getDistinctAttribute();
+update_distinct_attribute_1: |-
+  client.index("movies").setDistinctAttribute("movie_id");
```

Comment:
Same here, you should use `setDistinctAttribute` to set distinctAttribute in the settings and update the settings.